src/commands/status/status.ts

```diff
@@ -4,24 +4,22 @@ import prettyjson from 'prettyjson'
 
 import { chalk, error, exit, getToken, log, logJson, warn } from '../../utils/command-helpers.js'
 import BaseCommand from '../base-command.js'
+import { NetlifyLog, intro, outro } from '../../utils/styles/index.js'
 
 export const status = async (options: OptionValues, command: BaseCommand) => {
+  intro('status')
   const { api, globalConfig, site, siteInfo } = command.netlify
   const current = globalConfig.get('userId')
   const [accessToken] = await getToken()
 
   if (!accessToken) {
-    log(`Not logged in. Please log in to see site status.`)
-    log()
-    log('Login with "netlify login" command')
-    exit()
+    NetlifyLog.error(`Not logged in. Please log in to see site status.`)
```

Comment:
For this one, you might want to add a second parameter with `{exit: false}` so that line 16 doesn't immediately exit the process. Otherwise line 17 and 18 won't be shown. OR you can change it into a `NetlifyLog.warn` because it is more informative than actually going wrong :) 

I also think we don't need the additional empty one on line 17. I've so far removed all the empty `log()` ones I've seen.

Then on line 18 you can just use `NetlifyLog.message()` and then after that you could do `outro()`.

Reply:
Awesome, I needed this thank you

Reply:
![image](https://github.com/netlify/cli/assets/15314252/4ffc8d72-86bf-441d-acb3-8d8bacdf792a)

Reply:
done